lab_1/main.py

```diff
@@ -3,6 +3,9 @@
 Language detection
 """
 
+import re
+from typing import List, Dict, Any
```

Comment:
why do you need these imports?

Reply:
Hello, thank you, I'll try to do other functions.
What empty file should I remove?
As for imports, I've written import re, but the second import I haven't.